tests/models/bark/test_modeling_bark.py

```diff
@@ -51,6 +52,8 @@
         BarkSemanticModel,
     )
 
+set_seed(42)
```

Comment:
Do we require setting the seed now for the tests? Have we lost determinism somewhere?

Reply:
Nice catch, it was unrequired!

Reply:
Cool - will merge when the CI is green